src/actors/upgrade/vcs_repos/vcs_detection/check

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+
+import sys
+import json
+
+
+def load_inputs():
+    "Load all inputs and return dictionary with all input channels."
+    inputs = {}
+    if not sys.stdin.isatty():
+        input_data = sys.stdin.read()
+        if input_data:
+            inputs = json.loads(input_data)
+    return inputs
+
+
+def store_output(channel, data):
+    "Store the data into the expected channel (inside outputs object)."
+    # TODO: create class for input/output data? of course, this particular
+    # function works just for StringList now
+    outputs[channel] = [{"value": data}]
+    return
+
+
+def log_error(msg):
+    "Log error on stderr."
+    sys.stderr.write("Error: %s\n" % msg)
+
+
+def detect_repo(fname):
+    """
+    Check whether we found VCS repository or not.
+
+    In case we detected repository, return repository type (string). Otherwise
+    return None object.
+    """
+    for repo, suffix in vcs_map.items():
+        if fname.endswith("/%s" % suffix):
+            return repo
+    return None
+
+
+in_flist = "all_local_files"
```

Comment:
hmm the name doesn't say too much what it is and it looks like it should be "constant".

Reply:
Yes, it is constant. I was not sure about the name of input channel and didn't get better idea to write something small and reasonable.

Reply:
So if it's constant I would like to follow PEP8 convention
https://www.python.org/dev/peps/pep-0008/#constants